packages/bot-web-ui/src/assets/sass/load-modal.scss

```diff
@@ -0,0 +1,290 @@
+            line-height: 1.43;
+        }
+        &--selected {
+            background-color: var(--state-hover);
+        }
+        > * {
+            align-self: center;
+        }
+    }
+    &__item:hover {
+        background-color: var(--state-hover);
+        cursor: pointer;
+    }
+    &__preview {
+        width: 65%;
+
+        &-workspace {
+            padding: 15px;
+            position: relative;
+            border-radius: $BORDER_RADIUS;
+            border: solid 1px var(--border-normal);
+            height: 100%;
+            text-align: center;
+
+            .injectionDiv {
+                pointer-events: none;
+            }
+        }
+        &-controls {
+            padding: 7px 5px;
+            display: flex;
+            flex-direction: column;
+            position: absolute;
+            right: 16px;
+            bottom: 16px;
+            border-radius: 30px;
+            background-color: $color-grey-2;
+            z-index: 99;
+            box-shadow: 2px 2px 5px var(--shadow-menu);
+        }
+        &-icon {
+            margin: 5px;
+            cursor: pointer;
+        }
+    }
+    &__footer {
+        position: absolute;
+        bottom: 0;
+        right: 25px;
+    }
+}
+
+.local {
+    &__container {
+        padding: 25px;
+        text-align: center;
+        height: $modal_container_height;
+    }
+    &__dragndrop {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        border: dashed 3px var(--border-normal);
+        border-radius: $BORDER_RADIUS;
+        padding: 50px;
+        height: 100%;
+    }
+    &__title {
+        margin-top: 24px;
+        font-size: var(--text-size-s);
+    }
+    &__desc {
+        margin-top: 25px;
+        font-size: var(--text-size-xs);
+    }
+    &__icon {
+        margin: 0 auto;
+    }
+    &__upload {
+        margin: 10px auto 0;
+        width: fit-content;
+    }
+    &__preview {
+        height: 100%;
+
+        &-title {
+            text-align: left;
+            font-size: var(--text-size-s);
+            font-weight: bold;
+            line-height: 1.43;
+        }
+        &--hidden {
+            display: none;
+        }
+        &-workspace {
+            height: 100%;
+            padding: 15px;
+            margin-top: 8px;
+            border-radius: $BORDER_RADIUS;
+            border: solid 1px var(--border-normal);
+            position: relative;
+
+            .injectionDiv {
+                pointer-events: none;
+            }
+        }
+        &-close {
+            background-image: radial-gradient(at right top, $color_white 60%, transparent 90%);
+            position: absolute;
+            padding: 25px;
+            border-bottom-left-radius: 50%;
+            right: 0;
+            top: 0;
+            z-index: 99;
+            cursor: pointer;
+        }
+    }
+    &__footer {
+        position: absolute;
+        bottom: 0;
+        right: 25px;
+    }
+}
+
+.gd {
```

Comment:
- Could you scope this under `.load`? It'll prevent collision with other elements that have the same class.
- Could you use full words rather than generic names, e.g. `google-drive`?

Reply:
yeah sure.